src/main/java/teammates/ui/controller/InstructorFeedbackResponseCommentAddAction.java

```diff
@@ -1,128 +1,51 @@
-        if (showCommentTo != null && !showCommentTo.isEmpty()) {
-            String[] showCommentToArray = showCommentTo.split(",");
-            for (String viewer : showCommentToArray) {
-                feedbackResponseComment.showCommentTo.add(FeedbackParticipantType.valueOf(viewer.trim()));
-            }
-        }
-        feedbackResponseComment.showGiverNameTo = new ArrayList<>();
-        if (showGiverNameTo != null && !showGiverNameTo.isEmpty()) {
-            String[] showGiverNameToArray = showGiverNameTo.split(",");
-            for (String viewer : showGiverNameToArray) {
-                feedbackResponseComment.showGiverNameTo.add(FeedbackParticipantType.valueOf(viewer.trim()));
-            }
-        }
-
-        FeedbackResponseCommentAttributes createdComment = new FeedbackResponseCommentAttributes();
-        try {
-            createdComment = logic.createFeedbackResponseComment(feedbackResponseComment);
-            logic.putDocument(createdComment);
-        } catch (InvalidParametersException e) {
-            setStatusForException(e);
-            data.errorMessage = e.getMessage();
-            data.isError = true;
-        }
-
-        if (!data.isError) {
-            statusToAdmin += "InstructorFeedbackResponseCommentAddAction:<br>"
-                           + "Adding comment to response: " + feedbackResponseComment.feedbackResponseId + "<br>"
-                           + "in course/feedback session: " + feedbackResponseComment.courseId + "/"
-                           + feedbackResponseComment.feedbackSessionName + "<br>"
-                           + "by: " + feedbackResponseComment.giverEmail + " at "
-                           + feedbackResponseComment.createdAt + "<br>"
-                           + "comment text: " + feedbackResponseComment.commentText.getValue();
-        }
-
-        data.comment = createdComment;
-        data.commentId = commentId;
-        data.showCommentToString = StringHelper.toString(createdComment.showCommentTo, ",");
-        data.showGiverNameToString = StringHelper.toString(createdComment.showGiverNameTo, ",");
-        data.instructorEmailNameTable = bundle.instructorEmailNameTable;
-        data.question = logic.getFeedbackQuestion(feedbackQuestionId);
-        data.sessionTimeZone = session.getTimeZone();
+    @Override
+    protected String getUserEmailForCourse() {
+        return logic.getInstructorForGoogleId(courseId, account.googleId).email;
+    }
 
-        return createShowPageResult(Const.ViewURIs.INSTRUCTOR_FEEDBACK_RESPONSE_COMMENTS_ADD, data);
+    @Override
+    protected FeedbackSessionResultsBundle getDataBundle(String userEmail) throws EntityDoesNotExistException {
+        return logic.getFeedbackSessionResultsForInstructor(feedbackSessionName, courseId, userEmail);
     }
 
+    @Override
+    protected void appendToStatusToAdmin(FeedbackResponseCommentAttributes feedbackResponseComment) {
```

Comment:
Note: no need to rename methods in classes specifically related to comments (such as this one).